pkg/signing/signing.go

```diff
@@ -322,23 +324,27 @@ func Verify(cpath, keyServiceURI string, id uint32, isGroup bool, authToken stri
 		if err != nil {
 			// if theres a error, thats proboly becuse we dont have a local key
 
-			// download the key
-			sylog.Infof("Downloading key: %s...", fingerprint[24:])
-			netlist, err := sypgp.FetchPubkey(fingerprint, keyServiceURI, authToken, noPrompt)
-			if err != nil {
-				return fmt.Errorf("could not fetch public key from server: %s", err)
-			}
-			sylog.Verbosef("key retrieved successfully!")
-
-			block, _ := clearsign.Decode(data)
-			if block == nil {
-				return fmt.Errorf("failed to parse signature block")
-			}
-
-			// verify the container
-			signer, err = openpgp.CheckDetachedSignature(netlist, bytes.NewBuffer(block.Bytes), block.ArmoredSignature.Body)
-			if err != nil {
-				return fmt.Errorf("signature verification failed: %s", err)
+			if !localVerify {
+				// download the key
+				sylog.Infof("Downloading key: %s; not in keyring of trust.", fingerprint[24:])
```

Comment:
I think it's more accurate to have `sylog.Infof("Key with ID %x not found in local cache, searching for it via server %s", fingerprint[24:], keyServiceURI)`

Reply:
I shortened the message a bit, I know @gmkurtzer will say thats too long,

```
INFO:    Key with ID F1E47D55A7F3A56C not found in local keyring, downloading from keystore...
```

*It still could be shorter...*